src/scripts/preprocessing/kutuzov/rus_preprocessing_udpipe.py

```diff
@@ -13,7 +13,9 @@
 import wget
 import re
 from pathlib import Path
-from ufal.udpipe import Model, Pipeline
+## uncomment to import udpipe
```

Comment:
There should be an option to download these dependencies if the config is set to use udpipe rather than treetagger / mystem.  Otherwise lemmatization via udpipe is blocked indefinitely.